src/RestaurantButton.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+const RestaurantButton = ({ handler, message }) => {
+  return (
+    <button onClick={handler} className="btn btn-primary restBut">
```

Comment:
Well done with extracting the button and giving it an `onClick` handler🙂 I think it would make more sense to call the prop with the actual function name - `orderOne` instead of `handler`. It's more intuitive if the function name starts with a verb. Also I don't think that passing message is necessary as you already have order type in your Order component.